rust/arrow/src/ffi.rs

```diff
@@ -193,6 +205,20 @@ fn to_datatype(format: &str) -> Result<DataType> {
         "ttm" => DataType::Time32(TimeUnit::Millisecond),
         "ttu" => DataType::Time64(TimeUnit::Microsecond),
         "ttn" => DataType::Time64(TimeUnit::Nanosecond),
+
+        // Note: The datatype null will only be created when called from ArrowArray::buffer_len
+        // at that point the child data is not yet known, but it is also not required to determine
+        // the buffer length of the list arrays.
+        "+l" => DataType::List(Box::new(Field::new(
+            "",
```

Comment:
There isn't a single expected name for list children, but I'd prefer we use "item" as that's also used in C++. I think some name is better than an empty name

Reply:
check